blockchain/for-companies/how-to-run-liberland-company.md

```diff
@@ -0,0 +1,132 @@
+## Running Liberland registered companies
+So you have decided to pioneer liberty and decentralization by opening a company and doing business in Liberland.
+
+This article will explain how to create a company or open a subsidiary of another company, 
+how to handle payments either in Liberland itself or internationally or on-chain,
+how to sign and enforce legally binding contracts in Liberland or international jurisdictions, 
+and how to issue stock or other assets and list on the Liberland stock market.
+
+You will need to spend between 10 and 250 LLD for the process.
+
+[Browse existing companies](www.blockchain.liberland.org/home/registries/allCompanies)
+
+[Browse existing Contracts](www.blockchain.liberland.org/home/contracts/overview)
+
+### Choosing personal or company account
+Any person, wallet address of account can create a company in on the Liberland blockchain, however to be officially registered within the nation
+of Liberland it needs to satisfy some minimal requirements.
+
+The first decision is choosing if you want to open a company under your own personal account or create a new company account.
+
+We recommend creating a new company account (wallet address) and doing the process with the company account to keep finances and eventual
+contract obligations separate from you personally. However, depending on the nature of the business you might want to choose to open it under
+your personal account, especially if you are the only physical person in the company.
+
+It is easy to switch between many accounts at [blockchain.liberland.org](blockchain.liberland.org) in the menu wallet selector.
+
+![wallet_selector_list](../media/wallet-selector-list.png)
+
+Whichever account you choose, update your identity, especially display name and contact info in the profile page.
```

Comment:
Change the first two sentences to:
"After selecting an account, ensure to update your identity details, including your display name and contact information on your profile page. This step is crucial as it enables others to easily transfer tokens to your address by searching for your display name through the wallet at [blockchain.liberland.org](www.blockchain.liberland.org/home/wallet)."